### ISSUE TITLE
Accessibility: Missing `tabindex="0"` on clickable elements (Ele Custom Skin Pro)

### ISSUE THREAD

**Issue by jakebrightfire** [dudaster/ele-custom-skin#472]
When using a Post widget on the page and turning on the "Make entire post a link?" setting, there is an accessibility error with the output.

Here is the accessibility error:

> Interactive elements such as links, buttons and form fields should all be navigable using the keyboard by either using a focusable element (a, button, input, etc.) or including the "tabindex" attribute that equals to "0".

It is flagging the `<div class="esc-link-wrapper">` element, which should have a `tabindex="0"` attribute added to it. This `<div>` is located in `ele-custom-skin-pro/skin/skin-custom.php` line 312 in the `ECS_after_render_post_header` callback function.
